### PR TITLE
Rearranging actions

### PR DESCRIPTION
**Pull request by weerd**
This PR separates out Reportback related actions into its own file for easier organization. Not sure if this is ideal, but wanted to take a crack at it and see if it looks like it could help keep actions a bit more manageable!

I tested submitting a Reportback and all seemed to work fine, which is a good sign.

### CODE REVIEW

File: resources/assets/containers/ReportbackUploaderContainer.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { connect } from 'react-redux';
 import ReportbackUploader from '../components/ReportbackUploader';
-import { submitReportback, addToSubmissionsList } from '../actions';
+import { submitReportback, addToSubmissionsList } from '../actions/reportbackActions';
```
Comment (deadlybutter) [inline resources/assets/containers/ReportbackUploaderContainer.js]: Can you `export * from './reportbackActions';` inside the `actions/index` -- this way we only ever have to require a single file for actions

File: resources/assets/actions/reportbackActions.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
+/*
+ * Action names: import these constants to dispatch an event
+ * without having hardcoded strings all about.
+ */
+export const REQUESTED_REPORTBACKS = 'REQUESTED_REPORTBACKS';
```
Comment (DFurnes) [inline resources/assets/actions/reportbackActions.js]: I wonder if we should still have one file for constants so we can be ensure we don't have any overlapping names.

File: resources/assets/containers/ReportbackUploaderContainer.js (hunk 1 above)
Comment (weerd) [inline resources/assets/containers/ReportbackUploaderContainer.js]: I had that originally but I think you have to give it an alias, like:

```
export * as ReportbackActions from './reportbackActions';
```
Which means when importing they'd be props on a `ReportbackActions` object, right? But I may be confused...

File: resources/assets/actions/reportbackActions.js (hunk 2 above)
Comment (weerd) [inline resources/assets/actions/reportbackActions.js]: hmmm, yeah. Maybe that's what the `index.js` file is for, holding all the constants? :man_shrugging:

File: resources/assets/containers/ReportbackUploaderContainer.js (hunk 1 above)
Comment (deadlybutter) [inline resources/assets/containers/ReportbackUploaderContainer.js]: Try it? Not sure

Comment (deadlybutter) [inline resources/assets/containers/ReportbackUploaderContainer.js]: @weerd If that doesn't work, maybe something like this

```js
import * as reportbackActions from './reportbackActions';
export reportbackActions;
```

Comment (DFurnes) [inline resources/assets/containers/ReportbackUploaderContainer.js]: Yeah, I think you should be able to do multiple `export * from`s. I wonder what the error messaging looks like if you end up with naming collisions there.

File: resources/assets/actions/reportbackActions.js (hunk 2 above)
Comment (DFurnes) [inline resources/assets/actions/reportbackActions.js]: Yeah, that seems like a smart idea. `index.js` is the sorta like the table of contents then.

**Comment by weerd** [comment]
K! Thanks for the input. I'll rearrange a bit more based on your suggestions and see how it goes. I'll keep it to just the RB related actions for now.

**Comment by weerd** [comment]
@DFurnes @deadlybutter thanks for the suggestions. I was able to switch it to export all the functions from `reportbackActions.js` via `actions/index.js`, which makes it nicer to just needing to import the single `./actions` like you mentioned @deadlybutter

**Comment by weerd** [comment]
Lemme know if this looks cool and I can update the rest as well in following commits. :)

**Comment by deadlybutter** [comment]
Oh shoot thats what your comment was haha

**Comment by weerd** [comment]
Yeah, I can do that in this PR, just wanted to pick a direction first :)

**Pull request merged**